servers/physics/collision_solver_sat.cpp

```diff
@@ -98,7 +98,7 @@ static void _generate_contacts_edge_edge(const Vector3 *p_points_A, int p_point_
 
 	Vector3 c = rel_A.cross(rel_B).cross(rel_B);
 
-	//if ( Math::abs(rel_A.dot(c) )<_EDGE_IS_VALID_SUPPORT_TRESHOLD ) {
+	//if ( Math::abs(rel_A.dot(c) )<_EDGE_IS_VALID_SUPPORT_THRESHOLD ) {
```

Comment:
It would be better to just remove this line.

Reply:
Updated..